docs/framework/configure-apps/file-schema/windows-identity-foundation/identityconfiguration.md

```diff
@@ -4,89 +4,95 @@ ms.date: "03/30/2017"
-|-------------|-----------------|  
-|[\<caches>](../../../../../docs/framework/configure-apps/file-schema/windows-identity-foundation/caches.md)|Registers the caches used for session tokens and token replay detection. Can be specified at the service-level or on a security token handler collection. Optional.|  
-|[\<certificateValidation>](../../../../../docs/framework/configure-apps/file-schema/windows-identity-foundation/certificatevalidation.md)|Controls the settings that token handlers use to validate certificates. Can be specified at the service-level or on a security token handler collection. Optional.|  
-|[\<claimsAuthenticationManager>](../../../../../docs/framework/configure-apps/file-schema/windows-identity-foundation/claimsauthenticationmanager.md)|Registers a claims authentication manager for the incoming claims. Optional.|  
-|[\<claimsAuthorizationManager>](../../../../../docs/framework/configure-apps/file-schema/windows-identity-foundation/claimsauthorizationmanager.md)|Registers a claims authorization manager for the incoming claims. Optional.|  
-|[\<claimTypeRequired>](../../../../../docs/framework/configure-apps/file-schema/windows-identity-foundation/claimtyperequired.md)|Specifies the set of required claims for incoming security tokens. Optional.|  
-|[\<securityTokenHandlers>](../../../../../docs/framework/configure-apps/file-schema/windows-identity-foundation/securitytokenhandlers.md)|Specifies a collection of security token handlers. Zero or more collections of security token handlers can be specified. Optional.|  
-|[\<tokenReplayDetection>](../../../../../docs/framework/configure-apps/file-schema/windows-identity-foundation/tokenreplaydetection.md)|Enables token replay detection and specifies the expiration time for tokens. Can be specified at the service-level or on a security token handler collection. Optional.|  
-  
-### Parent Elements  
-  
-|Element|Description|  
-|-------------|-----------------|  
-|[\<system.identityModel>](../../../../../docs/framework/configure-apps/file-schema/windows-identity-foundation/system-identitymodel.md)|Provides configuration for enabling Windows Identity Foundation (WIF) options in applications.|  
-  
-## Remarks  
- Multiple identity configurations may be defined, each with a unique name. The behavior is as follows:  
-  
-1.  If no `<identityConfiguration>` element is specified. A default identity configuration is created at runtime and populated with default values.  
-  
-2.  If a single `<identityConfiguration>` element is specified. It is the default identity configuration. It does not matter whether it is named or unnamed.  
-  
-3.  If multiple `<identityConfiguration>` elements are specified. The unnamed element specifies the default identity configuration. It is recommended that when you specify multiple `<identityConfiguration>` elements, one of them should be unnamed.  
-  
+
+Specifies service-level identity settings.
+
+ \<system.identityModel>
```

Comment:
This change removes the new line between the two. 
![image](https://user-images.githubusercontent.com/12971179/53622835-fe1ade00-3baf-11e9-826f-6733bb5e34b7.png)

Maybe do this?

```suggestion
 \<system.identityModel>\
```